src/index.html

```diff
@@ -21,7 +21,427 @@
     />
   </head>
 
-  <body>
-    <h1>Catalog</h1>
+  <body class="page">
+    <header class="header">
+      <a
+        href="#"
+        class="header__img"
+      >
+        <img
+          src="./images/logo.png"
+          alt="Moyo"
+        />
+      </a>
+
+      <nav>
+        <ul class="header__nav-list">
+          <li>
+            <a
+              href="#apple"
+              class="header__nav-link is-active"
+            >
+              Apple
+            </a>
+          </li>
+
+          <li>
+            <a
+              href="#samsung"
+              class="header__nav-link"
+            >
+              Samsung
+            </a>
+          </li>
+
+          <li>
+            <a
+              href="#smartphones"
+              class="header__nav-link"
+            >
+              Smartphones
+            </a>
+          </li>
+
+          <li>
+            <a
+              href="#laptops-Computers"
+              class="header__nav-link"
+              data-qa="nav-hover"
+            >
+              Laptops & Computers
+            </a>
+          </li>
+
+          <li>
+            <a
+              href="#gadgets"
+              class="header__nav-link"
+            >
+              Gadgets
+            </a>
+          </li>
+
+          <li>
+            <a
+              href="#tablets"
+              class="header__nav-link"
+            >
+              Tablets
+            </a>
+          </li>
+
+          <li>
+            <a
+              href="#photo"
+              class="header__nav-link"
+            >
+              Photo
+            </a>
+          </li>
+
+          <li>
+            <a
+              href="#video"
+              class="header__nav-link"
+            >
+              Video
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main class="main">
+      <div
+        class="card"
+        data-qa="card"
+      >
+        <img
+          src="./images/imac.jpeg"
+          alt="iMac-icon"
+          class="card__image"
+        />
+
+        <h2 class="card__title">
+          APPLE A1419 iMac 27" Retina
+          <br />
+          5K Monoblock (MNED2UA/A)
+        </h2>
+
+        <p class="card__product-code">Product code: 195434</p>
+
+        <div class="stars">
+          <div class="stars stars--4">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+
+          <p class="stars__review">Reviews: 5</p>
+        </div>
+
+        <div class="card__price">
+          <div class="card__price card__price--name">Price:</div>
+          <div class="card__price card__price--value">$2,199</div>
```

Comment:
Use the semantic tag for text everywhere
```suggestion
          <p class="card__price card__price--name">Price:</p>
          <p class="card__price card__price--value">$2,199</p>
```